### ISSUE TITLE
simplify archive_uploads

### ISSUE THREAD

**Issue by TheExGenesis** [TheExGenesis/community-archive#109]
Instead of tracking which archive upload everything came from, let's only care about a user's latest upload - keep only one profile per user, remove archive_upload column from most tables, etc

**Comment by ri72miieop** [comment]
I agree with the removal of the tracking of the original archive, but I think it would be beneficial to have N records of the profile per user. In a way a profile change marks an era, people usually don't change it that often, so it can be seen as a boundary setting information. It adds a certain texture to that account.

For example, I could see this info as being updated by a browser extension in the future. There was a little tool some time ago that used the location to help people find moots that are near you, we could also do something similar if we collect this info and have it as updated as possible. Now you could argue that having only the most recent information would be sufficient, which, for this use-case, is valid. But I still think that having N profiles per user is valuable. A different use-case where this would be necessary: show a historically accurate thread (as in, the pfp is the same that it was at the time of posting, instead of being the current one).
